src/bin/rgb21.rs

```diff
@@ -160,3 +162,9 @@ fn main() -> Result<(), String> {
 
     Ok(())
 }
+
+fn parent_validator(parent_id: &str) -> Result<(), String> {
+    let _ = ContractId::from_str(parent_id).expect("Invalid parent ID");
```

Comment:
`expect` seems a bit too harsh here, could you please return an error? something as: `Err("Parent ID must be a valid contract ID".to_string())`